runtime/src/main/kotlin/kotlin/native/Text.kt

```diff
@@ -9,52 +9,103 @@ package kotlin.native
 /**
  * Converts an UTF-8 array into a [String]. Replaces invalid input sequences with a default character.
  */
+@Deprecated("Use decodeToString or decodeZeroTerminatedToString instead")
```

Comment:
`ReplaceWith(...)` is not provided here to push user to choose the right option mindfully.

Reply:
I think it's ok to provide an automatic replacement to the new function with the same behavior, if the migration to the other alternative is easy enough.

Reply:
+1